content/en/docs/reference/glossary/identity.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Identity
+---
+
+An identity is a unique name backed by key material, for
```

Comment:
Same comment about key material, that isn't the important part necessarily.

Maybe if we want to call that out we can say:

An identity is a unique service account name within a namespace, typically available through a cryptographically signed mechanism such as:

Reply:
Good call, @vadimeisenbergibm 

@rcaballeromx I think this (or some form of it) could probably just be used as our glossary definition for identity.  WDYT?

Reply:
@nmittler @vadimeisenbergibm Agreed. I'll use the content in the security concept to craft a definition for the next update. Thanks.